packages/start/README.md

```diff
@@ -66,6 +66,122 @@ const [theme, setTheme] = createUserTheme("cookieName", {
 theme(); // => "light" | "dark"
 ```
 
+
+## Examples
+
+### `root.tsx`
+
+This is the main entry file for SolidStart demonstrating usage of the `createUserTheme` function.
+- Initializes the theme with the `createUserTheme` function.
+- Toggles between the "dark" and "light" themes using a button.
+- Sets the `data-theme` attribute adding compatibility between `createUserTheme` and libraries like [DaisyUI](https://daisyui.com/).
+- Includes a clean implementation of theming through a `Body` class.
+- Provides additional examples on using variant classes to style components.
+
+```tsx
+// @refresh reload
+import { Suspense } from "solid-js";
+import {
+  A,
+  Body,
+  ErrorBoundary,
+  FileRoutes,
+  Head,
+  Html,
+  Meta,
+  Routes,
+  Scripts,
+  Title,
+} from "solid-start";
+import Counter from "./components/Counter";
+import "./root.css";
+import { createUserTheme } from "./start_primitive/index";
+export default function Root() {
+  const [theme, setTheme] = createUserTheme("dark-light-theme", "dark");
+  const toggleTheme = () => setTheme(currentTheme => currentTheme === "dark" ? "light" : "dark");
+  return (
+    // Set data-theme when using libraries like Daisyui:
+    <Html lang="en" data-theme={theme()}>
+      <Head>
+        <Title>SolidStart - Bare</Title>
+        <Meta charset="utf-8" />
+        <Meta name="viewport" content="width=device-width, initial-scale=1" />
+      </Head>
+      <Body
+        classList={{
+          // Apply theme as css class to body:
+          [theme()]: true,
```

Comment:
This could be simplified to
```tsx
class={theme()}
```

Reply:
I agree although I intentionally wrote it this way because of all the Solid code I see where people begin stacking into class like so:
```tsx
<Component class={`${theme()} text-style other-style`} />
```

This doc provided a good opportunity to show `classList` and doubly so using a computed property. With that said, I think you're right and `Body` should use `class` but I'd like to keep `classList` as a commented out alternative unless you're against it.